src/portfolios/portfolio/components/TaskOrder/TaskOrder.vue

```diff
@@ -15,11 +15,23 @@
             Add a new task order
           </v-btn>
         </div>
-        <TaskOrderCard :taskOrders="taskOrderData" :showDetails.sync="showTaskOrderDetails"/>
+        <TaskOrderCard
+          :isHistory="false"
+          :taskOrders="taskOrderData"
+          :showBuild.sync="showBuildPage"
+          :showDetails.sync="showDetails"
+          :selectedTaskOrder.sync="selectedTaskOrder"
+        />
       </div>
-      <div v-if="showTaskOrderDetails">
+      <div v-if="showBuildPage">
```

Comment:
remove all references to `BuildPage` and the component file(s) after discussing 8040 with Tony